corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
performance-wise i think it wouldn't matter. doing truncate..insert just makes it more explicit i think, and makes it easier to make changes to the process